projects/angular-split/src/lib/component/split.component.ts

```diff
@@ -628,11 +637,6 @@ export class SplitComponent implements AfterViewInit, OnDestroy {
       }
     })
 
-    this.snapshot.allInvolvedAreasSizePercent = [
```

Comment:
@beeman As far as I am aware this will always be 100. Anyway with wildcard it creates a bug so I removed it as seems to be redundant.